fork-clone-workflow.md

```diff
@@ -49,13 +49,14 @@ cd /opt/repos
 ```
 * Clone your fork using the `git clone` command. You can find the URL of your fork on the main [gihub](git-glossary.md#github)
 page of your fork. Click on the `Clone or download` button and a popup will show you the URL to use if you want to
-`Clone with SSH`. The SSH option is the easiest option to use but requires that you have confiured configured your
-[github](git-glossary.md#github) account with your public SSH key. As an alternative, you may select `Use HTTPS`. 
-The HTTPS option uses a different URL and will require you to supply your [github](git-glossary.md#github)
-account password each time you access your fork. The SSH option does not require you to supply your [gihub](git-glossary.md#github)
-when accessing your fork.
+`Clone with HTTPS`. There is also a `Clone with SSH` option which requires that you have confiured configured your
```

Comment:
> There is also a `Clone with SSH` option which requires that you have confiured configured your github account with your public SSH key. See Cloning using SSH for more information on using the `Clone with SSH` option to access remote repositories.

Can we move this to the end of the section/paragraph?

Reply:
Yes. I will update the branch in my fork that is referenced in the pull request. I think that will update the pull request.